mempool/mempool.go

```diff
@@ -110,6 +110,20 @@ func (mem *Mempool) SetLogger(l log.Logger) {
 	mem.logger = l
 }
 
+// Stop closes the underlying WAL file and then discards it.
+// After Stop has been invoked, any further writes will not be relayed to disk.
+func (mem *Mempool) Stop() error {
```

Comment:
let's include cmn.BaseService into Mempool and implement OnStart/OnStop. this is a standard across our repos. what do you think?

Reply:
@melekes I personally don't have any opinions about that. I was just requesting for a Stop method as the lack of it is cramping up my fuzzing tests and giving false positives allover the place. However, sure, I can implement that.